orchestration/dagster_orchestration/hca_manage/common.py

```diff
@@ -1,11 +1,24 @@
 import argparse
+from dataclasses import dataclass
 import sys
 from typing import NoReturn
 
 from data_repo_client import ApiClient, Configuration, RepositoryApi
 
 from hca_orchestration.contrib.google import default_google_access_token
 
+JobId = str
```

Comment:
this should have been changed to a subclass upstream - did you rebase recently?

Reply:
I saw that it was a class, but since it is just a string, I figured this was simpler/cleaner

Reply:
I covered this briefly during my review, but there's a difference in behavior here - if you just set it as an alias of `str`, then they're interchangeable, so passing a string that isn't a `JobId` to a function that accepts a `JobId` won't raise any alarms to mypy.

If you make it a subclass (we should have that somewhere), it'll ensure that functions that accept/return `JobId` can't be passed a standard string.

Reply:
WOW today I learned. Will change it back